src/config.c

```diff
@@ -303,3 +304,27 @@ bool config_to_color(const char* text, argb_t* color)
 
     return false;
 }
+
+bool config_to_translucent_color(const char* text, argb_t* color)
```

Comment:
I think we could merge `config_to_color` and `config_to_translucent_color`.
If alpha is not present - just set it as `0xff`.

Reply:
Alpha is set to `0xff` when not present in `config_to_translucent_color`, the reason I made a new function here is because fonts render incorrectly when the alpha is not `0x00` like would be the case in `config_to_color`.

Seperate functions:
![image](https://github.com/artemsen/swayimg/assets/91640048/d20ae476-025e-4d53-9ebc-0002d0e96256)

Merged with `0xff` alpha:
![image](https://github.com/artemsen/swayimg/assets/91640048/d0130ec8-c1fc-4ba9-b3ef-b62339a07e15)

I would just make the default fill for no alpha `0x00`, but we have no way of distinguishing an intentionally transparent color from an opaque one in that case.